pkg/envoy/server/health.go

```diff
@@ -7,7 +7,7 @@ func (s *Server) Liveness() bool {
 
 // Readiness is the Kubernetes readiness probe handler.
 func (s *Server) Readiness() bool {
-	return s.ready
+	return true
```

Comment:
This doesn't feel right; why are we effectively removing our readiness probe?

Reply:
There were no cases were s.ready was false prior to the server being started and able to handle requests, and we never switch it to false in any cases, so it was an unnecessary appendage.

Reply:
We defaulted the field to true? That feels like a bug....would prefer not to propagate it if possible

Reply:
essentially there's no not ready case. The server turns on and it's ready. There was never any downstream checks to perform

Reply:
i've gone ahead and removed the health check anyways, to just leverage the health.SimpleHandler